images/dotcms/ROOT/srv/templates/dotcms/OVERRIDE/WEB-INF/classes/db.properties

```diff
@@ -31,10 +25,14 @@ connectionTestQuery=SELECT 1
 
 username={{ .Env.PROVIDER_DB_USERNAME }}
 password={{ .Env.PROVIDER_DB_PASSWORD }}
+## Pool Min and Max connections
+minimumIdle=10
```

Comment:
We were missing the minimumIdle setting for our DB pool, making dotCMS request 200 connections on startup.